pysparkle/pysparkle/data_quality/README.md

```diff
@@ -0,0 +1,73 @@
+# Usage
+
+```bash
+pysparkle data-quality --help
+pysparkle data-quality --config-path "data_quality/silver_dq.json"
+```
+
+`config-path` is a Path to a JSON config inside an Azure Blob container. The container name
+for storing configurations is `config` and it is defined in the configuration set in the pysparkle
+package.
+
+Azure Blob storage account name is expected to be set in an environment variable as explained
+in the main README file. 
+
+
+# JSON Configuration File for Great Expectations
+This section describes the structure of the JSON configuration file used in our system.
+The configuration is defined using Python's Pydantic library for data validation.
+
+Here is the description of the main elements:
+
+1. `gx_directory_path`: Path to the Great Expectations metadata store.
+2. `dataset_name`: Name of the dataset that is being processed.
+3. `datasource_config`: List of datasource configurations where each configuration contains the following fields:
+   - `datasource_name`: Name of the data asset, e.g., table or file name.
+   - `datasource_type`: Source system type, e.g., table, parquet, csv. 
```

Comment:
(Assuming I've understood this right) I think it might be useful to add a bit more detail / examples to clarify what values are required, for either:
 - table : e.g. this refers to a hive metastore table, example of what to use for datasource_name & datasource_location
 - other file types : this needs to be a valid data source type that spark can read (e.g. csv, parquet), and for these a full path is required

Reply:
Updated slightly, the details about table have been added to the data_location parameter.